src/api/user/InsertUser.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+
+
+function InsertUser(token, id, keycloakParsed, callback) {
+    if (token) {
+        const user = {
+            data: {
+                id: id,
+                type: "users",
+                attributes: {
+                    firstName: keycloakParsed.given_name,
+                    lastName: keycloakParsed.family_name,
+                    email: keycloakParsed.email,
+                    // orchid: null,
```

Comment:
@samleeflang Do you mean the whole attributes object or just the orcid/organization id? Because the first name, last name and email could make use of the already existing data in the Keycloak token